clients/cobol-lsp-vscode-extension/src/services/util/FSUtils.ts

```diff
@@ -84,26 +27,24 @@ export function getURIFromResource(resource: string): urlUtil.URL[] {
  * @param copybookFolders list of folders from where to search the copybook
  * @param extensions list of possible copybooks extensions
  */
-export function searchCopybookInWorkspace(
+export function searchCopybookInExtensionFolder(
```

Comment:
I think that neither of these names is really appropriate...

Reply:
any suggestions ? maybe `searchCopybookInInternalFolder` ?

Reply:
Not, sure the function does too much stuff.